101-team-services-project-create/README.md

```diff
@@ -0,0 +1,9 @@
+# Deploy a new Team Services (VSTS) account and project.
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2F101-team-services-project-create%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+This template allows you to deploy a new Team Services (VSTS) account and a new project.
```

Comment:
From our brand guidelines, this line should say "This template allows you to deploy a new **Visual Studio Team Services** (VSTS) account and a new project". We are trying to clean up the inconsistent naming we have everywhere. The new guideline is to first introduce the product as "Visual Studio Team Services (VSTS)" and then use the abbreviation "VSTS" after that, and not to use the abbreviation "Team Services" at all. There a few instances of that in this PR, but I'm not flagging them all. Thanks for helping us out with this!